hack/charts/cluster-api-operator/templates/bootstrap.yaml

```diff
@@ -52,4 +52,7 @@ spec:
     {{- end }}
 {{- end }}
 {{- end }}
+{{- if hasKey (default (dict) $.Values.deploymentOverride) "bootstrap" }}
+  {{ .Values.deploymentOverride.bootstrap | toYaml | nindent 2 }}
```

Comment:
This isn't necessarily a deployment override, or? It allows to add custom stuff to the provider CR, which could be anything? Maybe the word 'deployment' is confusing here.
(I came here when looking for a way to add a `manifestPatches` key to the provider to set resource requests for the provider's deployment resource).